### ISSUE TITLE
About training, validating and testing

### ISSUE THREAD

**Issue by martisaju** [AlexeyAB/darknet#3507]
Hi, 
I am training Yolov3-tiny network to detect my own objects. I have a validation set and a test set but I am not sure how to use them in order to evaluate and tune my network. Should I put the validation set folder on data/valid, with the valid.txt referring to these images and then train the network? What about the test set? How can I check the network accuracy once it is trained with the test images?

Thank you so much.

**Comment by jjehl** [comment]
Training set : is used to train the network. Your network will always be good (after a long training) on your training set. So the testing set is to test on other no-trained images to be sure your network can not only detect the images of your training set. Darknet is a relatively complex program and you should be able to read some c code to understand how it works. There are different technique to evaluate your network (mAP or more simple things you could find in darknet.c).

**Comment by martisaju** [comment]
Thanks @jjehl . My issue is that I validate my network by using the training set (giving me very high mAP values) but I realized that I should do it in a different and independent set: the validation set. So my question is: how has to be it performed? Do I have to put the validation set folder at data/valid path  with the valid.txt file referring to these images?

Thanks
